functions/Remove-SqlOrphanUser.ps1

```diff
@@ -89,19 +92,19 @@ Will remove from all databases the user OrphanUser EVEN if exists their matching
 	)
 
     DynamicParam { if ($SqlServer) { return Get-ParamSqlDatabases -SqlServer $SqlServer -SqlCredential $SqlCredential } }
-	
+
 	BEGIN
 	{
         Write-Output "Attempting to connect to Sql Server.."
```

Comment:
Write-Output will mix with the PSCustomobject.
The message strings need to be written to an alternative stream.